### DISCUSSION TITLE
How to get the SVG file from a Spectrum icon

### DISCUSSION THREAD

**Discussion by martpet** [adobe/react-spectrum#4032] [Q&A] [answered]
I want to add a React Spectrum icon to a MapBox layer so I need to get the path to the SVG file. Is it possible?

**Comment by martpet** [comment, marked as answer]
Ok, I can get the SVG source from https://spectrum.adobe.com/page/icons

**Reply by snowystinger** [reply]
If you are an Adobe Developer, please request icons from the icons team.
